### PR TITLE
automate changelog

### LINKED ISSUE #30: Add changelog validator
See https://github.com/gitpod-io/gitpod/issues/4919

### PR DESCRIPTION
**Pull request by JanKoehnlein**
Fixes #30

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **arthursens** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @arthursens` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitbot/pull/31#issuecomment-912413478" title="Approved">JanKoehnlein</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **arthursens** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @arthursens` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: apply.sh [hunk 1]
```diff
@@ -6,13 +6,13 @@ cd config/prow
...
+kubectl create configmap config --from-file=config.yaml=config/config.yaml --dry-run=client -o yaml | kubectl replace configmap config -f -
 
 # note: ensure these config maps already exist
 
-kubectl create configmap projectmanager --from-file=projectmanager.yaml=config/projectmanager.yaml --dry-run -o yaml | kubectl replace configmap projectmanager -f -
-kubectl create configmap groundwork --from-file=groundwork.yaml=config/groundwork.yaml --dry-run -o yaml | kubectl replace configmap groundwork -f -
-kubectl create configmap customlabels --from-file=customlabels.yaml=config/customlabels.yaml --dry-run -o yaml | kubectl replace configmap customlabels -f -
-kubectl create configmap observer --from-file=observer.yaml=config/observer.yaml --dry-run -o yaml | kubectl replace configmap observer -f -
-kubectl create configmap willkommen --from-file=willkommen.yaml=config/willkommen.yaml --dry-run -o yaml | kubectl replace configmap willkommen -f -
+kubectl create configmap projectmanager --from-file=projectmanager.yaml=config/projectmanager.yaml --dry-run=client -o yaml | kubectl replace configmap projectmanager -f -
```
Comment (csweichel) [inline apply.sh:14]: is the dry-run in there on purpose?

Comment (JanKoehnlein) [reply apply.sh:14]: Yes, we get a warning otherwise. Is it wrong?

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 001f5ce2bb380c3c06b227d3dbfb5aaddbfc33ae</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitbot/pull/31#issuecomment-913739217" title="LGTM">csweichel</a>*, *<a href="https://github.com/gitpod-io/gitbot/pull/31#issuecomment-912413478" title="Approved">JanKoehnlein</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)~~ [csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
@JanKoehnlein: Updated the following 2 configmaps:
 * `plugins` configmap in namespace `prow` at cluster `default` using the following files:
   - key `plugins.yaml` using file `config/plugins.yaml`
 * `config` configmap in namespace `prow` at cluster `default` using the following files:
   - key `config.yaml` using file `config/config.yaml`


<details>

In response to [this](https://github.com/gitpod-io/gitbot/pull/31):

>Fixes #30


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**